climakitae/util/utils.py

```diff
@@ -685,3 +685,46 @@ def convert_to_local_time(data, selections):  # , lat, lon) -> xr.Dataset:
     selections.time_slice = (start, end)
 
     return sliced_data
+
+
+def add_dummy_time_to_wl(wl_da):
+    """
+    Replace the `[hours/days/months]_from_center` dimension in a DataArray returned from WarmingLevels with a dummy time index for calculations with tools that require a `time` dimension.
+
+    Parameters
+    ----------
+    wl_da : xarray.DataArray
+        The input Warming Levels DataArray. It is expected to have a time-based dimension which typically includes "from_center"
+        in its name indicating the time dimension in relation to the year that the given warming level is reached per simulation.
+
+    Returns
+    -------
+    xarray.DataArray
+        A modified version of the input DataArray with the original time dimension replaced by a dummy time series. The new dimension
+        will be named "time".
+
+    Notes
+    -----
+    - The function looks for the dimension name containing "from_center" to identify the time-based dimension.
+    - It supports creating dummy time series with frequencies of hours, days, or months, based on the prefix of the dimension name.
+    - The dummy time series starts from "2000-01-01".
+    """
+    # Adjusting the time index into dummy time-series for counting
+    # Finding time-based dimension
+    wl_time_dim = [dim for dim in wl_da.dims if "from_center" in dim][0]
```

Comment:
This primarily assumes only one dimension will be called `[something]_from_center`